### DISCUSSION TITLE
Implement admin useAsTitle for Array field items

### DISCUSSION THREAD

**Discussion by katywings** [payloadcms/payload#1104] [Feature Requests & Ideas]
While fiddling around with the fields and collections I was curious, if I could use the value of an array item field as the collapsible title of the item. Currently the title for array items is pretty much hard coded in: https://github.com/payloadcms/payload/blob/4900fa7/src/admin/components/forms/field-types/Array/index.tsx#L277. Do you know any way to customize the array item title without copying the whole Array form?

Unless there is another solution, what do you think about an `useAsTitle` setting for array field items?

**Comment by katywings** [comment]
Nevermind, sorry, you can delete this discussion :see_no_evil: there is already #808

**Reply by jmikrut** [reply]
Ha, ok. Yep. This is on the radar for sure and would be great to have. I have detailed notes on #808 so for anyone coming in, reference that discussion.

Thanks Katy!
